examples/pylint/E0202_method_hidden.py

```diff
@@ -0,0 +1,7 @@
+class Example(object):
```

Comment:
You can disable this check. I'd prefer an error on masking a method (not for right now, just in general).